with-splash-screen/package.json

```diff
@@ -1,4 +1,9 @@
 {
+  "scripts": {
+    "start": "expo start",
+    "android": "expo start --android",
+    "ios": "expo start --ios"
+  },
```

Comment:
```suggestion
```

To serve this example as a template, none of the examples in this repository has `"scripts"` declared in **package.json**.